extensions/amp-byside-content/0.1/amp-byside-content.js

```diff
@@ -122,32 +150,55 @@ export class AmpBysideContent extends AMP.BaseElement {
    * @override
    */
   preconnectCallback(onLayout) {
-    if (this.iframeSrc_) {
-      this.preconnect.url(this.iframeSrc_, onLayout);
+    if (this.origin_) {
+      this.preconnect.url(this.origin_, onLayout);
     }
   }
 
   /** @override */
   buildCallback() {
     this.webcareId_ = user().assert(
         this.element.getAttribute('data-webcare-id'),
-        'The data-webcare-id attribute is required for <' + TAG_ + '> %s',
-        this.element);
+        'The data-webcare-id attribute is required for <%s> %s',
+        TAG_,
+        this.element
+    );
 
     this.label_ = user().assert(
         this.element.getAttribute('data-label'),
-        'The data-label attribute is required for <' + TAG_ + '> %s',
-        this.element);
+        'The data-label attribute is required for <%s> %s',
+        TAG_,
+        this.element
+    );
 
     this.webcareZone_ = (this.element.getAttribute('data-webcare-zone') ||
-		DEFAULT_WEBCARE_ZONE_);
+        DEFAULT_WEBCARE_ZONE_);
     this.channel_ = (this.element.getAttribute('data-channel') || '');
     this.lang_ = (this.element.getAttribute('data-lang') || DEFAULT_LANG_);
     this.fid_ = (this.element.getAttribute('data-fid') || '');
 
+    this.isResizable_ = this.element.hasAttribute('resizable');
```

Comment:
we really like to avoid sub-areas of page that are vertically scrollable. It created bad UX. May I ask why scrollable iframe is preferred to resized version in some situations in your case?

Reply:
Removed resizable custom property, always assume non scrollable.